administration/src/util/translations/de.json

```diff
@@ -561,7 +562,9 @@
     "regionNotUnique": "Den Akzeptanzstellen konnte keine eindeutige Region zugewiesen werden. Bitte wenden Sie sich an den IT-Support",
     "pdfCreationError": "Etwas ist schiefgegangen beim Erstellen der PDF.",
     "csvCreationError": "Etwas ist schiefgegangen beim Erstellen der CSV.",
-    "freinetAgencyNotFound": "Diese Freinetagentur existiert nicht."
+    "freinetAgencyNotFound": "Diese Agentur konnte in Freinet nicht gefunden werden.",
+    "freinetMultiplePersonError": "Es wurden mehrere Personen mit diesen Daten gefunden. Bitte aktualisieren Sie die Daten in Freinet-Online manuell.",
+    "freinetApiNotReachableError": "Es ist keine Verbindung zu Freinet-Online möglich. Bitte versuchen Sie es zu einem späteren Zeitpunkt erneut. Bitte aktualisieren Sie die Daten in Freinet-Online manuell."
```

Comment:
@connium 
I think here we even need a better description

Reply:
It wont send any request to freinet If the card creation got aborted. so I will go with second one...